src/jl_uv.c

```diff
@@ -1085,6 +1085,24 @@ JL_DLLEXPORT int jl_queue_work(work_cb_t work_func, void *work_args, void *work_
     return 0;
 }
 
+JL_DLLEXPORT int jl_uv_update_timer_start(uv_loop_t* loop, jl_value_t* jltimer,
+                                          uv_timer_t* uvtimer, uv_timer_cb cb,
+                                          uint64_t timeout, uint64_t repeat)
+{
+    JL_UV_LOCK();
+    int err = uv_timer_init(loop, uvtimer);
+    if (err) {
+        JL_UV_UNLOCK();
+        return err;
+    }
+
+    jl_uv_associate_julia_struct((uv_handle_t*)uvtimer, jltimer);
+    uv_update_time(loop);
+    err = uv_timer_start(uvtimer, cb, timeout, repeat);
```

Comment:
similarly, this can't actually fail either, we can just `abort` if it returns non-zero